### PR TITLE
python-api: make work on python 3, packaging stuff...

### PR DESCRIPTION
**Pull request by mlyle**

### CODE REVIEW

**Comment by mlyle** [comment]
Minimally tested this-- with logs-- but need to try telemetry sessions too.

**Comment by mlyle** [comment]
Now it distributes the bins, too.

File: python/setup.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+        # Pick your license as you wish (should match "license" above)
+        'License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)',
+
+        # Specify the Python versions you support here. In particular, ensure
+        # that you indicate whether you support Python 2, Python 3 or both.
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7'
```
Comment (tracernz) [inline python/setup.py]: No python 3?

**Comment by tracernz** [comment]
`pip -U install dronin`? :stuck_out_tongue:  :100:

File: python/setup.py (hunk 1 above)
Comment (mlyle) [inline python/setup.py]: Will fix for future uploads :)

**Comment by mlyle** [comment]
or dronin[all] to bring in dependencies (Except PyQt4).

**Comment by tracernz** [comment]
Do you want anything specific tested with this? I should be able to tell pip to install this from disk right?

**Comment by mlyle** [comment]
it's already pushed to pypi, so you can just do pip install dronin[all]

I'm happy with this, though I have not tested the full suite of capabilities.  In particular I haven't tested serial FC comms.  But there's going to be further active dev and this is a worthwhile step along the way (and appropriately publishes the code of what we have on pypi)

**Comment by mlyle** [comment]
And now fixed up to work with serial and network telemetry well.

**Pull request merged**